### PR TITLE
Add support for Scroll pHAT HD from Pimoroni

### PR DESCRIPTION
**Pull request by dglaude**
Tested on CLUE using BIT:2:Pi to plug the Scroll pHAT HD.
Used the same orientation as the CharliePlex LED matrix Bonnet.
Added this option in all example, and enabled it on the Adafruit Scroll Text example.

PS: Should also work for the new Scroll HAT Mini.

### CODE REVIEW

**Comment by dglaude** [comment]
Demo of the code in action: https://twitter.com/DavidGlaude/status/1238974062560411649?s=20

File: adafruit_is31fl3731.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
 import time
 from micropython import const
 
-__version__ = "0.0.0-auto.0"
+__version__ = "2.5.3"
```
Comment (ladyada) [inline adafruit_is31fl3731.py]: revert this

File: examples/is31fl3731_text_example.py [hunk 2]
```diff
@@ -11,7 +11,9 @@
 # uncomment line if you are using Adafruit 16x9 Charlieplexed PWM LED Matrix
 #display = adafruit_is31fl3731.Matrix(i2c)
 # uncomment line if you are using Adafruit 16x9 Charlieplexed PWM LED Matrix
-display = adafruit_is31fl3731.CharlieBonnet(i2c)
+#display = adafruit_is31fl3731.CharlieBonnet(i2c)
+# initial display using Pimoroni Scroll Phat HD LED 17 x 7
+display = adafruit_is31fl3731.ScrollPhatHD(i2c)
```
Comment (ladyada) [inline examples/is31fl3731_text_example.py]: revert, dont make scrollphat the default

**Review by ladyada — CHANGES_REQUESTED** [review]

**Comment by dglaude** [comment]
Requested change done.

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
ok thanks!

**Pull request merged**